Cargo.toml

```diff
@@ -898,7 +898,7 @@ memory-db = { version = "0.33.0", default-features = false }
 merkleized-metadata = { version = "0.5.0" }
 merlin = { version = "3.0", default-features = false }
 messages-relay = { path = "bridges/relays/messages" }
-metered = { version = "0.6.1", default-features = false, package = "prioritized-metered-channel" }
+metered = { git = "https://github.com/paritytech/orchestra", branch = "alexggh/router_priority_messages", default-features = false, package = "prioritized-metered-channel" }
```

Comment:
Not intended for merging, will be removed once a new orchestra release with https://github.com/paritytech/orchestra/pull/87 happens.